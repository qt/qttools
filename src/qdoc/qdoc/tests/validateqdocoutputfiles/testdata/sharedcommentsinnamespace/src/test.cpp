// Copyright (C) 2024 The Qt Company Ltd.
// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0

#include "test.h"

/*! \module Mod */
/*! \namespace Test
    \inmodule Mod
*/
namespace Test {
/*!
    \fn void foo()
    \fn void bar()

    Two functions sharing a doc comment.
*/
}
