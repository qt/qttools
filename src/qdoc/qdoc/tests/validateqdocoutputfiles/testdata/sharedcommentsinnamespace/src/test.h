// Copyright (C) 2024 The Qt Company Ltd.
// SPDX-License-Identifier: LicenseRef-Qt-Commercial OR GPL-3.0-only WITH Qt-GPL-exception-1.0
#pragma once

namespace Test {
    void foo(){}
    void bar(){}
}
