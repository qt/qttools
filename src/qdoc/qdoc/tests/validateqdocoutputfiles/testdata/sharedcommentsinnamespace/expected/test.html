<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8">
<!-- test.cpp -->
  <title>Test Namespace | SharedCommentsInNamespace</title>
</head>
<body>
<div class="sidebar">
<div class="toc">
<h3 id="toc">Contents</h3>
<ul>
<li class="level1"><a href="#functions">Functions</a></li>
<li class="level1"><a href="#details">Detailed Description</a></li>
</ul>
</div>
<div class="sidebar-content" id="sidebar-content"></div></div>
<h1 class="title" translate="no">Test Namespace</h1>
<div class="table"><table class="alignedsummary requisites" translate="no">
<tr><td class="memItemLeft rightAlign topAlign"> Header:</td><td class="memItemRight bottomAlign"> <code translate="no">#include &lt;Test&gt;</code></td></tr>
</table></div>
<h2 id="functions">Functions</h2>
<div class="table"><table class="alignedsummary" translate="no">
<tr><td class="memItemLeft rightAlign topAlign"> </td><td class="memItemRight bottomAlign"><b></b></td></tr>
<tr><td class="memItemLeft rightAlign topAlign"> void </td><td class="memItemRight bottomAlign"><b><a href="test.html#bar" translate="no">bar</a></b>()</td></tr>
<tr><td class="memItemLeft rightAlign topAlign"> void </td><td class="memItemRight bottomAlign"><b><a href="test.html#foo" translate="no">foo</a></b>()</td></tr>
</table></div>
<!-- $$$Test-description -->
<div class="descr">
<h2 id="details">Detailed Description</h2>
</div>
<!-- @@@Test -->
<div class="func">
<h2>Function Documentation</h2>
<!-- $$$ -->
<div class="fngroup">
<h3 class="fn fngroupitem" translate="no" id="bar"><span class="type">void</span> Test::<span class="name">bar</span>()</h3><h3 class="fn fngroupitem" translate="no" id="foo"><span class="type">void</span> Test::<span class="name">foo</span>()</h3></div>
<p>Two functions sharing a doc comment.</p>
<!-- @@@ -->
</div>
</body>
</html>
